lib/helper.js

```diff
@@ -130,11 +132,11 @@ function withContext(Component, context) {
 
     // Do not use hasOwnProperty, we need all keys from the entire prototype chain
     for (var key in context) {
-        childContextTypes[key] = React.PropTypes.any;
+        childContextTypes[key] = PropTypes.any;
     }
 
-    return React.createClass({
-        displayName: (Component.displayName || 'Component') + ':withContext',
+    return createReactClass({
+        displayName: (Component.displayName || 'UnnamedComponent') + ':withContext',
```

Comment:
I think you want to change the [test assertion](https://github.com/brucewpaul/jsx-test/blob/6f7589c68b13140d4387659c33a33c7a2958a318/test/withContext.test.js#L59) right? Because this could break people upstream that test for this name by chance.